docs/core/reference/versioning.md

```diff
@@ -0,0 +1,72 @@
+# Chain Core Versioning
+
+## Summary
+
+Chain Core versioning falls into two categories: **package** versioning and **network** versioning.
+
+Package versioning applies to the pieces of software that work together to participate in a blockchain network. For example, the Chain Core Mac app and the Java SDK.
+
+Network versioning applies to the network API - the interface through which different Chain Cores on a network comunicate to exchange transactions and blocks. Any breaking changes to the network API interface or Chain protocol are reflected by a change to the network version.
+
+## Package Versioning
+The various packages of Chain Core each use a three number versioning scheme - **x.y.z**. The first two numbers indicate compatablity between packages. For example, if you are a running a version of the Chain Core Mac app whose first two numbers are **1.1**, then you should also use a version of an SDK whose first two numbers are also **1.1**.
```

Comment:
One thing that @oleganza made sure to do in his proposal, that it seems like we've lost here, is to give a name (he called it "product version") to those first two numbers. Right now we have to awkwardly refer to them as "the first two numbers".